spec.ttl

```diff
@@ -0,0 +1,595 @@
+@prefix : <> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix doap: <http://usefulinc.com/ns/doap#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix spec: <#> .
+@prefix test-description: <http://www.w3.org/2006/03/test-description#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix vs: <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+spec:
+  a owl:Ontology ;
+  rdfs:label "Spec Terms"@en ;
+  dcterms:description "Spec Terms provides classes and properties that can be used to describe any significant unit of information in technical reports."@en ;
```

Comment:
```suggestion
  rdfs:label "Spec Terms ontology"@en ;
  dcterms:description "The Spec Terms ontology provides classes and properties that can be used to describe any significant unit of information in technical reports."@en ;
```

Reply:
Thank Ted. I say the following without wishing us getting drawn into a discussion on whether what's herein is a vocabulary or an ontology.

As I see it, Spec Terms is not detailed or complex enough or intended to be an ontology in its proper sense. It is intended as a simple vocabulary. I'm well-aware that the document includes OWL (but primarily RDFS and SKOS) however don't think that the human-readable content should express that.

Would you object to not mentioning "ontology" in the human-readable text?